src/lib/server/database.ts

```diff
@@ -0,0 +1,19 @@
+import type {Project} from '$lib/types/project'
+
+const db: Project[] = []
+
+export const getProjects = () => [...db]
+
+export const createProject = (name: string, configuration: unknown) => {
+    const id = `${getProjects().length + 1}`
```

Comment:
```suggestion
    const id = `${getProjects().length}`
```
Наверное более логично)